docs/source/whats_new.rst

```diff
@@ -18,6 +18,7 @@ Develop branch
 Enhancements
 ~~~~~~~~~~~~
 
+- Add VirtualReality BrainInvaders dataset (:gh:`358` by `Gregoire Cattan`_)
```

Comment:
Could please add also your name also at the end of the file? This way, your name is linked correctly on the website.